scripts/redcap/import_mr_sessions

```diff
@@ -88,7 +89,15 @@ redcap_token_path = os.path.join( os.path.expanduser("~"), '.server_config/redca
 redcap_token_file = open( redcap_token_path, 'r' )
 redcap_token = redcap_token_file.read().strip()
 
-redcap_project = redcap.Project( 'https://ncanda.sri.com/redcap/api/', redcap_token, verify_ssl=False)
+try:
+    redcap_project = redcap.Project( 'https://ncanda.sri.com/redcap/api/',
```

Comment:
Ok, I have no rights to install software in my machine, please do it at your convenience.